src/app/services/events/events.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from '../../../environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class EventsService {
+
+  constructor(private http: HttpClient) { }
+
+  getEvents(): Promise<Event[]> {
```

Comment:
One way could be to set the `getEventbriteEvents()` and `getGoogleCalendarEvents()` as private methods and create a public method `getEvents()` which calls both and merges them into one. For that, we'll need to have a matching routine to determine if the item from Google Calendar API is the same as the item provided in Eventbrite API.

Reply:
Like a Facade?
getEventFrom('Google'): Promise<Event[]> {
  if ...
    return func1();
  else if ...
    return fun2();
}

Im not sure whether this is necessary since we only got two event functions